geoportal/geoportailv3_geoportal/__init__.py

```diff
@@ -476,6 +476,9 @@ def main(global_config, **settings):
     config.add_route('upload_vt_style', '/uploadvtstyle')
     config.add_route('delete_vt_style', '/deletevtstyle')
     config.add_route('get_vt_style', '/getvtstyle')
+    config.add_route('upload_permalink_style', '/uploadpermalinkstyle')
+    config.add_route('delete_permalink_style', '/deletepermalinkstyle')
+    config.add_route('get_permalink_style', '/getpermalinkstyle')
```

Comment:
Why do we need to introduce the  `...permalink...` entry points for the v4 additionally to the other ones?

Reply:
- permalink storage (for permalink sharing and maputnik style editor) no sync with tile server raster renderer
- tile server ( only upload and delete) raster tiles are rendered for 3D and printing